src/librustc_save_analysis/dump_visitor.rs

```diff
@@ -32,16 +33,20 @@ use syntax::print::pprust::{
 };
 use syntax::ptr::P;
 use syntax::source_map::{Spanned, DUMMY_SP, respan};
+use syntax::walk_list;
 use syntax_pos::*;
 
-use {escape, generated_code, lower_attributes, PathCollector, SaveContext};
-use json_dumper::{Access, DumpOutput, JsonDumper};
-use span_utils::SpanUtils;
-use sig;
+use crate::{escape, generated_code, id_from_def_id, id_from_node_id, lower_attributes,
```

Comment:
`id_from_def_id` and `id_from_node_id` are widely used in this module, and different functions with the same name are not used, so they are added to imports.